airbyte-integrations/connectors/destination-bigquery/src/test-integration/java/io/airbyte/integrations/destination/bigquery/typing_deduping/BigQueryStandardInsertsRawOverrideTypingDedupingTest.java

```diff
@@ -0,0 +1,9 @@
+package io.airbyte.integrations.destination.bigquery.typing_deduping;
+
+public class BigQueryStandardInsertsRawOverrideTypingDedupingTest extends AbstractBigQueryTypingDedupingTest {
```

Comment:
question: I see this pattern a lot in the test suite.  Do we need to run _all_ the tests again, or would it be sufficient just to run one of them, just making sure a sync doesn't blow up and the schema we expect was used.  

Looking at this test now, I think it would pass if the override wasn't in use at all, and we used the default?

Reply:
ah yeah that's a good point... I guess if we have the test explicitly override `getRawDataset` to return literal `overridden_raw_dataset` then we'd catch it (because the test would try to dump raw records from a nonexistent table). I'll make that change.

IMO there's enough interesting tests (i.e. overwrite, incremental dedup, `namespace=null` all seem useful to run for every config) that it's worth just running the entire suite, but curious if you feel otherwise